src/actions/actions_makeshift.py

```diff
@@ -23,12 +23,23 @@ class MakeshiftAction:
     MakeshiftAction("hover", "showHover", "editor.action.showHover"),
     MakeshiftAction("inspect", "showDebugHover", "editor.debug.action.showDebugHover"),
     MakeshiftAction(
-        "quick fix", "showQuickFix", "editor.action.quickFix", restore_selection=True
+        "quick fix",
+        "showQuickFix",
+        "editor.action.quickFix",
+        restore_selection=True,
+        post_command_sleep_ms=100,
```

Comment:
Is a sleep time necessary for quick fix? It seems to work fine for me without it.

Reply:
Try saying "quick fix air enter" to select the first suggestion

Reply:
I did that multiple times and it seemed to work fine for so I'm was just wondering if it was different for you.

Reply:
Yeah wasn't working for me so I added it. Could be an OS thing. You mention you're also able to use command server while quick pick is up, which I cannot. The quick pick menu kinda looks like a native Mac menu so might behave differently